data/tcp-connector.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @title TCP Connector: Ping
+ * @difficulty intermediate
+ * @tags cli
+ * @run --allow-net <url>
+ * @resource {https://deno.land/api?s=Deno.listen} Deno.listen: API
+ * @resource {https://github.com/denoland/denobyexample/blob/main/data/tcp-listener.ts} Example TCP server that will accept connections
```

Comment:
Let's use the site relative path
```suggestion
 * @resource {/tcp-listener} Example TCP server that will accept connections
```